src/Makefile

```diff
@@ -73,7 +76,7 @@ all: $(TARGET)
 
 $(TARGET): $(OBJS)
 	@echo "  $(notdir $@)"
-	$(Q)$(CC) $(CFLAGS) $^ -o $@
+	$(Q)$(CC) $(CFLAGS) $^ $(LIBS) -o $@
```

Comment:
` $(LIBS)` should be removed from `CFLAGS` in line 70. It's the reason why the CI complained a failure.

Reply:
@ywc689 I also agree with you, but this is not introduced by current changes. This looks like a regression issue from previous commits. 

But can change for the community